### ISSUE TITLE
Camera-ready (RecSys Challenge 2026, paper 1275): organizer + reviewer feedback — due Aug 17

### ISSUE THREAD

**Issue by npatta01** [npatta01/music-crs-2026#211]
Camera-ready tracking issue for RecSys Challenge 2026 paper **1275** — *State-Driven Retrieval and Learned Re-Ranking for Conversational Music Recommendation* (accepted, Workshop Track, 2 × weak accept, reproducibility 4/4).

**Deadline: August 17, 2026 — strict, no extensions.** At least one author must also register by the same date for the paper to appear in the proceedings.

Source of the feedback: EasyChair notification `recsys2026-challenge@easychair.org`, 2026-08-10 (metareview + 2 reviews) and a follow-up 2026-08-11 ("Additional Camera-Ready Recommendations").

Related: PR #210 (`codex/recsys26-camera-ready`). **PR #210 is a formatting/TAPS-compliance PR only — it does not address any of the substantive reviewer points below**, and it introduces one citation error (see A3). Coverage assessment is inline.

---

## A. Organizer / logistics requirements

| # | Requirement | PR #210 |
|---|---|---|
| A1 | Latest ACM TAPS template; restore proceedings metadata | ✅ done — `acmart` 2.19 sigconf, `printacmref=false` / `\setcopyright{none}` removed, TikZ replaced with vector PDFs (TikZ/PGF is not on ACM's accepted package list), `\Description` on all 4 figures + 5 tables |
| A2 | Team name, code-repo link, achieved score clearly stated | ✅ already in the paper — "team npatta01", repo footnote, 0.3811 composite / 0.2537 nDCG@20 / 29-of-39 |
| A3 | Cite the 4 organizer-supplied references | ⚠️ **partial + wrong** — see below |
| A4 | All links fully working at submission time | ✅ re-verified 2026-08-15, all HTTP 200: challenge site, `github.com/npatta01/music-crs-2026`, `huggingface.co/datasets/Npatta01/music-crs-repro-2026`, `docs.lancedb.com` |
| A5 | Page limit: 4 content + 1 reference page; up to **5 content pages** allowed **only** to address reviewer comments | ⚠️ PDF is 5 pages, ~4.3 content pages — but the extra space is consumed by formatting changes, not reviewer responses. Either bring content back under 4 pages or actually spend the room on §B. |
| A6 | ACM eRights block (`\setcopyright`, `\acmDOI`, `\acmISBN`) | ⛔ blocked — placeholder comment only; needs the eRights email. `paper/main.pdf` on the branch is a **preflight artifact**, not the upload. |
| A7 | ORCID for every author, in the eRights form and the PDF | ❌ **missing** — no `\orcid{}` on any of the three authors in `main.tex` |
| A8 | Affiliations complete | ⚠️ PR sets all three to Independent Researcher, Hoboken, New Jersey, USA. Confirm this is actually correct for Semih and Tanwir. |
| A9 | Email the repo link to `recsys2026.music.crs@gmail.com` for the official challenge site | ❌ not done — out of scope for the PR, needs a human |
| A10 | Name the TAPS upload ZIP with ACM's proceedings acronym + assigned paper ID | ⛔ blocked on the TAPS email |

### A3 — citation problems (fix before upload)

**Bug:** `paper/references.bib` has the organizers' overview paper with *our* author names substituted:

```bibtex
@inproceedings{doh2026recsyschallenge,
  author = {Nidhin Pattaniyil and Semih Yagli and Tanwir Zaman},   % ← wrong
  title  = {{RecSys Challenge 2026}: Conversational Music Recommendation},
  ...
}
```

The organizers' supplied entry has `author = {Doh, Seungheon and Oramas, Sergio and Sguerra, Bruno and Bohra, Abhinav and Pomo, Claudio and Barile, Francesco}`. As committed, the bibliography misattributes someone else's paper to us. Must be restored verbatim.

**Present:** `doh2026recsyschallenge` (authors wrong), `choi2025talkplaydata` (committed under the key `talkplay`).

**Missing — add both:**

```bibtex
@article{doh2025talkplay,
  title = {TalkPlay: Multimodal Music Recommendation with Large Language Models},
  author = {Doh, Seungheon and Choi, Keunwoo and Nam, Juhan},
  journal = {arXiv preprint arXiv:2502.13713},
  year = {2025}
}

@inproceedings{doh2026llmjudge,
  author = {Doh, Seungheon and Sguerra, Bruno and Oramas, Sergio and Epure, Elena V. and Nam, Juhan},
  title = {{LLM-as-a-Judge} for Evaluating System Responses in Conversational Music Recommendation},
  booktitle = {Proceedings of the 20th ACM Conference on Recommender Systems},
  series = {RecSys '26},
  year = {2026},
  location = {Minneapolis, MN, USA},
  publisher = {Association for Computing Machinery}
}
```

`doh2026llmjudge` is directly relevant — the composite metric we report weights an LLM judge at 30%, and §Response generation discusses our offline judge being unreliable. It should be cited where the metric and the judge are introduced.

---

### Metareview summary
> Reviewers agree the system itself […] is not the contribution, the transparency is […] Points to address: unexplained technical details (fuzzy resolver, weight thresholds); model selection remains biased because design choices were made repeatedly on the same 8,000 dev turns; the LLM relabeling study must be framed strictly as exploratory alternative annotation (LLM judges arbitrated by a third LLM, no human gold, missing prompts/uncertainty); the RRF-vs-LambdaMART comparison needs a same-pool control; and the selected 4.70 response score is not an independent estimate.

- [ ] **B1 — Explain the fuzzy resolver (R1).** `main.tex` §State extraction currently says only "A fuzzy resolver grounds surface names to catalog IDs so downstream stages operate on identifiers." Add the actual matching mechanism and its failure mode (the Blind-B audit attributes 1 flagged turn to an entity-resolution miss).
- [ ] **B2 — Justify the label down-weight thresholds 0.3 / 0.6 (R1).** Currently stated as bare constants. Say whether they were tuned, guessed, or inherited, and on what.
- [ ] **B3 — Explain design choices generally (R1).** "Many of the technical design choices could be better explained" — e.g. why 11 branches, why pool depth 500, why RRF `k=60`, why the 0.6 tag-resolver cosine threshold.
- [ ] **B4 — State the model-selection bias explicitly (metareview + R2).** §In-sample already admits row-level leakage and overfit selection. The reviewers want the stronger, distinct point in the limitations: user-grouped out-of-fold removes row leakage *for a fixed pipeline*, but branch design, feature set, label weights, and hyperparameters were all chosen after repeatedly examining the same 8,000 dev turns, so no estimate here is an independent selection estimate; the Blind-B score is the only external check, and there is no blind component baseline.
- [ ] **B5 — Reframe the relabeling as exploratory alternative annotation (metareview + R2).** Two LLM judges, ~20% disagreement, a third LLM arbitrating, **no human gold sample**. The 57% fit-disagreement and 17% anchoring figures are *model judgments*, not estimates of true label error, and must be worded that way. Add or link: the judge prompts, judge inputs, the adjudication rule, per-axis agreement, and uncertainty.
- [ ] **B6 — Specify the "trained on relabeled data scored lower" claim (R2).** Missing the model, score, split, and evaluation target. Reviewer's point stands: if it was scored against the *original* labels, a decline is the expected outcome and does not invalidate the alternative annotation. State which target was used, or soften the claim.
- [ ] **B7 — Same-pool control for RRF vs LambdaMART (metareview + R2).** RRF fuses branch rankings to depth 1,000 while LambdaMART scores branch pools truncated to 500 — not a like-for-like comparison. Reviewer also asks for per-branch and union target recall, oracle nDCG over the pool, and the fraction of groups with no positive in the pool.
- [ ] **B8 — Ablation evidence / bi-encoder increment (R2).** No controlled ablations for typed state, branch families, label weighting, constraint guards, or response conditioning. The `b1_cos` increment (0.1970 → 0.2032, +0.0062) is reported without fold dispersion and was never tested on blind data. Add dispersion or explicitly scope the claim.
- [ ] **B9 — Response-score provenance (metareview + R2).** The 8-variant sweep 4.20 → 4.70 was selected on Blind-A leaderboard feedback because the offline judge was unreliable — so 4.70 is a *selected*, not independent, estimate and must be labelled as such. Also: the official results table reports Blind-A judge **4.20**, and it is unclear which configuration produced that row and which produced the Blind-B submission. Reconcile explicitly.

### Where the extra page should go
A5 grants a fifth content page **only** for reviewer responses. B4, B5, B6, B7 and B9 are the ones that need prose; B1–B3 are short clarifications. That is the intended use of the space, and it is currently unused.

---

## What PR #210 does and does not do

**Does (all verified against the diff):** `acmart` 2.16 → 2.19; restores ACM reference/rights block; TikZ → embedded-font vector PDFs (`figures/pipeline.pdf`, `figures/biencoder.pdf`); `\Description` on 4 figures + 5 tables; manual `thebibliography` → `references.bib` + `ACM-Reference-Format`; city/state on affiliations; `\shortauthors`; generative-AI disclosure in `\acks`; `paper/CAMERA_READY.md` with build instructions and checklist; rebuilt 5-page US-Letter PDF.

**Does not:** any of B1–B9. `main.tex` is +31/−89 lines and the additions are almost entirely `\Description` accessibility text — no reviewer-response content was written.

**Introduces:** the A3 citation-author bug. `paper/CAMERA_READY.md` documents it as intentional ("used the three author names printed in this paper, as instructed by the authors") — that instruction was a misreading of the organizers' email and should be reverted.

### Suggested sequencing given the Aug 17 deadline
1. Fix A3 (bib authors + 2 missing citations) and A7 (ORCID) — mechanical, minutes.
2. Write B1, B2, B3 — short clarifications.
3. Write B4, B5, B6, B9 — limitation/framing prose, no new experiments needed.
4. B7 and B8 need numbers we may or may not have offline; if a same-pool RRF control and pool-recall/oracle-nDCG figures can't be produced in time, say so explicitly in limitations rather than leaving the comparison as-is.
5. Then merge #210, paste the eRights block (A6), confirm A8, send A9, upload to TAPS.

**Comment by npatta01** [comment]
Partial progress pushed to PR #210 (`codex/recsys26-camera-ready`, now at 2e21319).

**A3 — done.** `doh2026recsyschallenge` author list restored to the organizers' (Doh, Oramas, Sguerra, Bohra, Pomo, Barile); it had this paper's own three names. Added and cited the two missing recommended references: `doh2025talkplay` (with the TalkPlayData 2 dataset in §1) and `doh2026llmjudge` (where the composite metric's judge term is defined). All four recommended references are now present and cited. `CAMERA_READY.md` corrected — it had recorded the wrong author names as intentional.

**A7 — partial.** ORCID added for Nidhin Pattaniyil (0009-0006-4594-3477). Semih Yagli and Tanwir Zaman still need to register at orcid.org; a TODO in `main.tex` marks where their IDs go. **Blocking the TAPS upload** — each author must register their own.

Verified on both commits: `lualatex → bibtex → lualatex → lualatex` builds clean, no undefined citations or LaTeX warnings, still 5 pages with references starting on page 5. Citation edits were zero net words, so nothing reflowed.

`paper/main.pdf` is deliberately **not** rebuilt in these commits — the local toolchain here is TeX Live 2025 / acmart 2.16 and would have downgraded the 2.19 build. It needs one rebuild on the 2026 toolchain at the same time the eRights block (A6) goes in.

Still open: A5 (page budget vs. reviewer responses), A6, A8, A9, A10, and all of B1–B9.
